resources/assets/js/bootstrap.js

```diff
@@ -0,0 +1,130 @@
+/* global window, document, $, autosize, Messenger, moment, livestamp, swal */
+import Vue from 'vue';
+import VueResource from 'vue-resource';
+
+// App config
+const Cachet = window.Cachet || {};
+
+Cachet.Globals = {
+  host: window.location.host,
```

Comment:
I was thinking we should follow airbnb javascript rules, what do you think? https://github.com/airbnb/javascript maybe include their eslint config